src/App.js

```diff
@@ -1,15 +1,15 @@
 import React from "react";
-import PDFViewer from "./components/PDFViewer";
-import FileUpload from "./components/FileUpload";
 
-import { test } from "./constants/testUrls";
+import "./styles/App.css";
+
+import CustomHeader from "./components/CustomHeader";
+import FileUpload from "./components/FileUpload";
 
 function App() {
   return (
-    <div className="App">
-      <h1>Hello there</h1>
+    <div className="wrapper">
+      <CustomHeader text="CleverReader" />
```

Comment:
Don't use magic words/numbers. If you are going to have statically defined names for different headers using same component, create a file which contains constants.
example:
type CleverReaderHeader = "CleverReader" in e.q. types.js
And then in App.js have <CustomHeader text=CleverReaderHeader>